geode-docs/managing/security/implementing_authentication.html.md.erb

```diff
@@ -23,151 +23,150 @@ Authentication lends a measure of security to a cluster
-        security-peer-auth-init=com.example.security.ServerAuthenticate.create
-        ```
-
-    - Implement the `getCredentials` method within the `AuthInitialize`
-    interface to acquire values for
-    the `security-username` and `security-password` properties
-    in whatever way it wishes.
-    It might look up values in a database or another external resource.
-
-Gateway senders and receivers communicate as a component of their
-server member.
-Therefore, the credential of the server become those of the gateway
-sender or receiver.
-
-## How a Client Cache Sets Its Credential
-
-In order to connect with a locator or a server that does authentication,
-a client will need to set its credential, composed of the two properties
+Choose one of two ways to set the server credentials:
+
+- Add settings to the server properties file, or
+- Implement the `AuthInitialize` interface for the server
+
+### <a id="authentication-setserverprops"></a>Add Settings to the Server Properties File
+
+Set `security-username` and `security-password` in the server's
+`gfsecurity.properties` file, which is read upon server startup.
+For example:
+
+``` pre
+security-username=admin
+security-password=xyz1234
+```
+The username and password are stored in the clear, so the
+`gfsecurity.properties` file must be protected by restricting access with
+file system permissions.
+
+### <a id="authentication-implementserverinterface"></a>Implement the AuthInitialize Interface for the Server
+
+To implement the `AuthInitialize` interface for the server, set the
+`security-peer-auth-init` property so that an object of the class that implements the `AuthInitialize`
+interface will be instantiated. There are two ways to do this:
+
+- You can set the `security-peer-auth-init` property to the fully-qualified class name that implements
+   the `AuthInitialize` interface as in the example
+
+  ``` pre
+  security-peer-auth-init=com.example.security.ServerAuthenticate
+  ```
+
+- You can set the `security-peer-auth-init` property to the fully-qualified method name of a method
+that instantiates an object of the class that implements the `AuthInitialize` interface as in the
+example
+
+  ``` pre
+  security-peer-auth-init=com.example.security.ServerAuthenticate.create
+  ```
+
+Implement the `getCredentials` method within the `AuthInitialize` interface to acquire values for
+the `security-username` and `security-password` properties in whatever way you wish.  For example,
+it might look up values in a database or another external resource.
+
+Gateway senders and receivers communicate as components of their respective server members. Therefore, the
+credentials of the server become those of the gateway sender or receiver.
+
+## <a id="authentication-client-set-creds"></a>How a Client Cache Sets its Credentials
+
+In order to connect with a locator or a server that performs authentication,
+a client must set its credentials, a username and password specified as the two properties
```

Comment:
again, it doesn't have to be username/password, it's whatever property generated by the `security-client-auth-init` class's getCredentials() method call.

Reply:
Thanks, @jinmeiliao Change incorporated.